src/main/java/org/jitsi/impl/protocol/xmpp/XmppProtocolProvider.java

```diff
@@ -288,6 +295,12 @@ synchronized private boolean doConnect()
             // rely on Smack's built-in retries, as it will be handled by
             // the RetryStrategy
             connection.removeConnectionListener(connListener);
+
+            ReconnectionManager reconnectionManager =
```

Comment:
shouldn't the equals sign be moved to the next line ?

Reply:
ups